v4/newrelic/segments.go

```diff
@@ -198,18 +216,59 @@ func (s *DatastoreSegment) End() {
 	if s.StartTime.isEnded() {
 		return
 	}
+
+	s.addAttributes(s.StartTime.Span.SetAttribute)
 	s.StartTime.Span.SetName(s.name())
 	s.StartTime.end()
 }
 
+func (s *DatastoreSegment) addAttributes(set func(string, interface{})) {
+	set("db.system", valOrUnknown(string(s.Product)))
+	set("db.statement", valOrUnknown(s.statement()))
+	set("db.operation", valOrUnknown(s.Operation))
+	set("db.collection", valOrUnknown(s.Collection))
```

Comment:
This one attribute is not defined in the opentelemetry specs. I decided to include it because we had this info on our datastore spans in v3 and I didn't want this information to be lost to our customers.